### ISSUE TITLE
[SA 1.4.7] Rotary Condensentrator Non-Existent

### ISSUE THREAD

**Issue by xBlizzDevious** [MJRLegends/Space-Astronomy-Feedback-#399]
I'm trying to build a Mekanism Fusion Reactor but the Rotary Condensentrator doesn't exist. The Wiki also states that a portable tank is required in the recipe but portable tanks don't exist either (Mekanism ones, at least).

I have checked the mekanism.cfg file and both are enabled.

**Comment by MJRLegends** [comment]
They do exist just craft them and you will see

**Comment by xBlizzDevious** [comment]
That's my issue - the recipes DO NOT exist. I can't see them in NEI and I can't craft them with the recipes from the wiki.

**Comment by MJRLegends** [comment]
What version of the mod pack you using

**Comment by MJRLegends** [comment]
Because they show for me

**Comment by xBlizzDevious** [comment]
1.4.7 through the FTB Launcher. I'm on a privately hosted server (by me).

I checked in a separate single player world too - no luck.

I did however, manage to find it in the creative tabs from which I can view the recipes - which the portable tank is different to the wiki and explains why I couldn't make it.

I have now successfully crafted the items but they're still not showing in NEI and they really should.

The ID for the Condensentrator is 1092:0 and the ID for the "Basic Fluid Tank" ie. Portable Tank is 1092:11 and they are both quite sneakily missing. All of the item between those values exist, however.
